src/phoenix/server/api/types/JSONSchema.py

```diff
@@ -0,0 +1,9 @@
+import strawberry
+from strawberry.scalars import JSON
+
+
+@strawberry.type
+class JSONSchema:
```

Comment:
this might be largely un-necessary

Reply:
But it might be nice to have a "container" for this in case there are non-json schemas?

Reply:
It may be valuable so we can union type for Json schema conforming json, vs non conforming json (schema-less)